inspector.ipxe

```diff
@@ -1,6 +1,5 @@
 #!ipxe
 
-
```

Comment:
needed whitespace churn?

Reply:
well, we have one already, two seemed like overkill :)